cmd/kola/spawn.go

```diff
@@ -57,6 +61,7 @@ func init() {
 	cmdSpawn.Flags().IntVarP(&spawnNodeCount, "nodecount", "c", 1, "number of nodes to spawn")
 	cmdSpawn.Flags().StringVarP(&spawnUserData, "userdata", "u", "", "file containing userdata to pass to the instances")
 	cmdSpawn.Flags().BoolVarP(&spawnDetach, "detach", "t", false, "-kv --shell=false --remove=false")
+	cmdSpawn.Flags().StringVar(&spawnOmahaPackage, "omaha-package", "", "add an update payload to the Omaha server, referenced by image version (e.g. 'latest')")
```

Comment:
Would it make sense to mirror the `updatepayload` functionality where you can either specify a direct link to the payload you want to test and only generate a payload from the `latest` image version if one is not provided?

Reply:
The default empty string here has a different semantics, it is used to signal that the user does not want to opt-in the whole omaha-package machinery. If the user specifies a version, then this will either re-use an existing payload or try to generate it first (if it doesn't exist).